scripts/dotnet-nuget3.cmd

```diff
@@ -0,0 +1,12 @@
+@Echo OFF
```

Comment:
I added it for debugging purposes and wanted to leave it in for a little bit. I can remove it though. It's useful as new commands light up to start testing them. I don't want to risk shipping it though.